lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -696,9 +696,8 @@ TypeLoweringVisitor::addArg(Operation *module, unsigned insertPt,
 
   SmallString<16> symtmp;
   StringRef sym;
-  bool oldArgHadSym = oldArg.sym && !oldArg.sym.getValue().empty();
-  if (oldArgHadSym) {
-    symtmp = (oldArg.sym.getValue() + field.suffix).str();
+  if (oldArg.sym) {
```

Comment:
(Will handling per-field symbols be needed here in the future?  There are outstanding PR's tackling lowering per-field symbols in lowering IIRC, just checking)

Reply:
Once GrandCentral gets lowered during LowerAnnotation, we will see field sensitive symbols so it would be necessary in the near future.